src/fsharp/InfoReader.fs

```diff
@@ -783,4 +814,67 @@ let PropTypOfEventInfo (infoReader: InfoReader) m ad (einfo: EventInfo) =
     let argsTy = ArgsTypOfEventInfo infoReader m ad einfo 
     mkIEventType g delTy argsTy
 
+let private RuntimeSupportsFeature (infoReader: InfoReader) m featureId =
+    let g = infoReader.g
+
+    let runtimeFeature =
+        match featureId with
+        | LanguageFeature.DefaultInterfaceMethodConsumption -> "DefaultImplementationsOfInterfaces"
+        | _ -> String.Empty
+
+    if String.IsNullOrWhiteSpace runtimeFeature then
+        true
+    else
+        match g.System_Runtime_CompilerServices_RuntimeFeature_ty with
+        | Some runtimeFeatureTy ->
+            infoReader.GetILFieldInfosOfType (Some runtimeFeature, AccessorDomain.AccessibleFromEverywhere, m, runtimeFeatureTy)
+            |> List.exists (fun ilFieldInfo -> ilFieldInfo.FieldName = runtimeFeature)
+        | _ ->
+            false
+
+[<Struct>]
+type LanguageFeatureSupport (runtimeError: exn option, langError: exn option) =
```

Comment:
Seems like an odd place to put this functionality, can it go in another file?

Reply:
Yea, we should put this in LanguageFeatures.fs.

Reply:
I still need to put it in another place.